tests/framework/extensions/psact/createdeployment.go

```diff
@@ -68,7 +69,7 @@ func CreateNginxDeployment(client *rancher.Client, clusterID string, psact strin
 			return false, err
 		}
 
-		if *deployment.Spec.Replicas == deployment.Status.AvailableReplicas && (psact == "" || psact == rancherPrivileged) {
+		if *deployment.Spec.Replicas == deployment.Status.AvailableReplicas && (psact == rancherPrivileged || psact == rancherBaseline) {
```

Comment:
is the null case still covered/important here?

Reply:
Honestly, I am not sure why it was changed like that to begin with. I didn't have that when I initially had PSACT support and not sure why we would need that even covered if PSACT isn't defined